src/modules/address-book/ModuleAddressBook.vue

```diff
@@ -201,7 +201,17 @@ export default {
             const reverseName = await this.nameResolver.resolveAddress(
               this.inputAddr
             );
-            this.resolvedAddr = reverseName.name ? reverseName.name : '';
+            if (!reverseName.name) {
+              reverseName.name =
+                (
+                  await getAddressInfo(
+                    toChecksumAddress(this.inputAddr),
+                    'https://ipfs.kleros.io'
```

Comment:
This doesn't need to be in connections?

Reply:
It seemed to work fine without it but I will add to connections